src/transformers/feature_extraction_utils.py

```diff
@@ -286,22 +286,24 @@ def from_pretrained(
 
         return cls.from_dict(feature_extractor_dict, **kwargs)
 
-    def save_pretrained(self, save_directory: Union[str, os.PathLike]):
+    def save_pretrained(self, save_directory: Union[str, os.PathLike], processor_class_str: Optional[str] = None):
         """
         Save a feature_extractor object to the directory ``save_directory``, so that it can be re-loaded using the
         :func:`~transformers.feature_extraction_utils.FeatureExtractionMixin.from_pretrained` class method.
 
         Args:
             save_directory (:obj:`str` or :obj:`os.PathLike`):
                 Directory where the feature extractor JSON file will be saved (will be created if it does not exist).
+            processor_class_str (:obj:`str`, `optional`):
+                Processor class to be saved in config.
```

Comment:
I have no idea how GitHub thinks he'll be able to merge this in the current Markdown docstring but you need a rebase and adapt that docstring here.

Reply:
Sorry will do!